ci/cron/daily-compat.yml

```diff
@@ -196,3 +196,61 @@ jobs:
         env:
           GCRED: $(GOOGLE_APPLICATION_CREDENTIALS_CONTENT)
       - template: ../daily_tell_slack.yml
+
+  - job: blackduck_scan
+    timeoutInMinutes: 1200
+    pool:
+      name: linux-pool
+      demands: assignment -equals default
+    steps:
+      - checkout: self
+      - bash: ci/dev-env-install.sh
+        displayName: 'Build/Install the Developer Environment'
+      - bash: ci/configure-bazel.sh
+        displayName: 'Configure Bazel'
+        env:
+          IS_FORK: $(System.PullRequest.IsFork)
+          # to upload to the bazel cache
+          GOOGLE_APPLICATION_CREDENTIALS_CONTENT: $(GOOGLE_APPLICATION_CREDENTIALS_CONTENT)        
+      - template: ../bash-lib.yml
+        parameters:
+          var_name: bash_lib
+      - bash: ./build.sh "_$(uname)"
+        displayName: 'Build'
+      - bash: |
+          set -euo pipefail
+          eval "$(./dev-env/bin/dade-assist)"
+          bash <(curl -s https://raw.githubusercontent.com/DACH-NY/security-blackduck/master/synopsys-detect) \
+          ci-build digital-asset_daml master \
+          --logging.level.com.synopsys.integration=DEBUG \
+          --detect.tools=BAZEL \
+          --detect.bazel.target=//... \
+          --detect.bazel.dependency.type=haskell_cabal_library \
+          --detect.notices.report=true \
+          --detect.report.timeout=1500
+        displayName: 'Blackduck Haskell Scan'
+        env:
+          BLACKDUCK_HUBDETECT_TOKEN: $(BLACKDUCK_HUBDETECT_TOKEN)
+      - bash: |
+          set -euo pipefail
+          eval "$(./dev-env/bin/dade-assist)"
+          bash <(curl -s https://raw.githubusercontent.com/DACH-NY/security-blackduck/master/synopsys-detect) \
+          ci-build digital-asset_daml master \
+          --logging.level.com.synopsys.integration=DEBUG \
+          --detect.npm.include.dev.dependencies=false \
+          --detect.excluded.detector.types=NUGET \
+          --detect.excluded.detector.types=GO_MOD \
+          --detect.yarn.prod.only=true \
+          --detect.python.python3=true \
+          --detect.tools=DETECTOR,BAZEL,DOCKER \
+          --detect.bazel.target=//... \
+          --detect.bazel.dependency.type=maven_install \
+          --detect.detector.search.exclusion.paths=language-support/ts/codegen/tests/ts,language-support/ts,/Users/brianhealey/g/daml/language-support/scala/examples/iou-no-codegen,language-support/scala/examples/quickstart-scala,docs/source/app-dev/bindings-java/code-snippets,docs/source/app-dev/bindings-java/quickstart/template-root,language-support/scala/examples/quickstart-scala,language-support/scala/examples/iou-no-codegen \
+          --detect.cleanup=false \
+          --detect.policy.check.fail.on.severities=MAJOR,CRITICAL,BLOCKER \
```

Comment:
Why do we only fail here but not on the Haskell dependencies in the first step?

Reply:
This is after part 2 which will return these failures for the entirety of things (inclusive of part 1)

Reply:
What exactly makes the second run picks up the results from the first one, i.e., how is information passed between them? I couldn’t find any docs for how this works.